tests/test_pagination.py

```diff
@@ -39,7 +47,7 @@ def test_run(self):
                                        for message in synced_records.get(stream).get('messages')
                                        if message.get('action') == 'upsert']
 
-                # verify records are more than page size so multiple page is working
+                # verify records are more than page size so multiple pages are working
```

Comment:
Can you chunk pages and validate unique values across all pages instead of the just the first 2?
For reference https://github.com/singer-io/tap-mixpanel/blob/2cf3a86231da042f76f4e3ef9fa8c23600b6f593/tests/tap_tester/test_all_fields_pagination.py#L134-L153

Reply:
Updated the code to validate unique values across all the pages instead of the first 2 pages.